db/migrate/20230125105529_create_post.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :CommentCounter
+      t.integer :LikesCounter
```

Comment:
- [OPTIONAL ] I highly recommend to change the names here to be comments_counter and likes_counter since this is best practice when naming the column in Ruby on rails so I think you can add new migration to change the name of the column🙏🏻💯✨✅✅🔥